.lintstagedrc

```diff
@@ -4,7 +4,7 @@
     "git add"
   ],
   "*.md": [
-    "prettier --prose-wrap always --write",
+    "prettier --no-config --prose-wrap always --write",
```

Comment:
This is so the readme doesn't get formatted with the same style as the source but stays as it is.

Reply:
interesting - what's the argument of not having the md formatted like everything else? sincerely curious

Reply:
I only made this change to preserve things the way they currently are.

Currently the JSON and the JS in the readme is not formatted similarly to the source code. For instance the source uses tabs for indentation while the code in the readme uses 2 spaces. The source uses `'` for quotes while the readme uses `"` in the JS in the bottom. This is because the config in `.lintstagedrc` uses two different set of arguments to Prettier.

If there isn't any particular reason for this then I can change it so they're both formatted the same.

Reply:
Let's just align the configs then for all file types